### PR TITLE
Minor peacekeeper tweaks.

### PR DESCRIPTION
**Pull request by icantthinkofanameritenow**
See the changelog. With the new description, there is no excuse for someone to be a shitter on Peacekeeper.

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -318,12 +318,12 @@
...
 			module_sprites["Black Knight"] = "securityrobot"
-			module_sprites["Bloodhound"] = "bloodhound"
+			module_sprites["Hound"] = "bloodhound"
 			module_sprites["Securitron"] = "securitron"
-			module_sprites["Marina-SC"] = "marinaSC"
+			module_sprites["Marina-PK"] = "marinaSC"
 			module_sprites["#9"] = "servbot-sec"
 			module_sprites["Kodiak"] = "kodiak-sec"
-			to_chat(src, "<span class='warning'><big><b>Just a reminder, by default you do not follow space law, you follow your lawset</b></big></span>")
+			to_chat(src, "<span class='warning'><big><b>Under ASIMOV, you are an enforcer of the PEACE and preventer of HUMAN HARM.You are expected to follow orders and prevent harm above all else. Space law means nothing to you.</b></big></span>")
```
Comment (Intigracy) [inline code/modules/mob/living/silicon/robot/robot.dm]: Space after HARM.

**Review by Intigracy — CHANGES_REQUESTED** [review]

**Review by icantthinkofanameritenow — COMMENTED** [review]
fixed the space.

**Comment by Ephx1** [comment]
That's very specific to Asimov, I'm not sure that's a good idea.

**Comment by icantthinkofanameritenow** [comment]
@Ephx1 
Asimov is the default, so it will be sufficient for roundstart barring adminbus law changes beforehand.  Even if they are changed, that last line doesn't leave any room for interpretation.

**Comment by TheFriggenSawbones** [comment]
Image of BLOODHOUND

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 2]
```diff
@@ -318,12 +318,12 @@
...
 			module_sprites["Black Knight"] = "securityrobot"
-			module_sprites["Bloodhound"] = "bloodhound"
+			module_sprites["Hound"] = "bloodhound"
 			module_sprites["Securitron"] = "securitron"
-			module_sprites["Marina-SC"] = "marinaSC"
+			module_sprites["Marina-PK"] = "marinaSC"
 			module_sprites["#9"] = "servbot-sec"
 			module_sprites["Kodiak"] = "kodiak-sec"
-			to_chat(src, "<span class='warning'><big><b>Just a reminder, by default you do not follow space law, you follow your lawset</b></big></span>")
+			to_chat(src, "<span class='warning'><big><b>Under ASIMOV, you are an enforcer of the PEACE and preventer of HUMAN HARM. You are expected to follow orders and prevent harm above all else. Space law means nothing to you.</b></big></span>")
```
Comment (Probe1) [inline code/modules/mob/living/silicon/robot/robot.dm]: Brevity is important.

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 3]
```diff
@@ -318,12 +318,12 @@
 			module_sprites["Basic"] = "secborg"
 			module_sprites["Blue Knight 2.0"] = "sleeksecurity"
 			module_sprites["Black Knight"] = "securityrobot"
-			module_sprites["Bloodhound"] = "bloodhound"
+			module_sprites["Hound"] = "bloodhound"
```
Comment (Probe1) [inline code/modules/mob/living/silicon/robot/robot.dm:321]: Why

**Review by Probe1 — DISMISSED** [review]

File: code/modules/mob/living/silicon/robot/robot.dm (hunk 3 above)
Comment (icantthinkofanameritenow) [reply code/modules/mob/living/silicon/robot/robot.dm:321]: Because "Bloodhound" sounds too aggressive. It's supposed to be peaceful.

Comment (Probe1) [reply code/modules/mob/living/silicon/robot/robot.dm:321]: Keeping the peace and being peaceful aren't the same thing.

Comment (icantthinkofanameritenow) [reply code/modules/mob/living/silicon/robot/robot.dm:321]: It still sounds like it's better suited to hunting criminals than it is at keeping the peace.

**Comment by SonixApache** [comment]
What if your lawset isn't asimov when you get built?

**Comment by icantthinkofanameritenow** [comment]
@SonixApache 
I removed the specifically asimov related parts, and any more changes would defeat the purpose of changing the notifier at all.

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 4]
```diff
@@ -318,12 +318,12 @@
...
 			module_sprites["Securitron"] = "securitron"
-			module_sprites["Marina-SC"] = "marinaSC"
+			module_sprites["Marina-PK"] = "marinaSC"
 			module_sprites["#9"] = "servbot-sec"
 			module_sprites["Kodiak"] = "kodiak-sec"
-			to_chat(src, "<span class='warning'><big><b>Just a reminder, by default you do not follow space law, you follow your lawset</b></big></span>")
+			to_chat(src, "<span class='warning'><big><b>You are expected to follow orders and prevent harm above all else. Space law means nothing to you- you are not a Security Officer.</b></big></span>")
```
Comment (SonixApache) [inline code/modules/mob/living/silicon/robot/robot.dm]: >You are expected to follow orders and prevent harm above all else.

Really only applies to asimov. Replace it with something like "You are expected to follow orders and keep the peace" or add a per-law clause, this is dumb.

**Comment by SonixApache** [comment]
👌

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 5]
```diff
@@ -323,7 +323,7 @@
 			module_sprites["Marina-PK"] = "marinaSC"
 			module_sprites["#9"] = "servbot-sec"
 			module_sprites["Kodiak"] = "kodiak-sec"
-			to_chat(src, "<span class='warning'><big><b>You are expected to follow orders and prevent harm above all else. Space law means nothing to you- you are not a Security Officer.</b></big></span>")
+			to_chat(src, "<span class='warning'><big><b>You are expected to follow your laws above all else. Space law means nothing to you- you are not a Security Officer.</b></big></span>")
```
Comment (Probe1) [inline code/modules/mob/living/silicon/robot/robot.dm]: You're still saying 1.  A mouthful and 2.  That space law doesn't apply when it does under robocop.

Comment (SonixApache) [reply code/modules/mob/living/silicon/robot/robot.dm]: Increase the warning size, add a sound (the poke sound from TS3 is a good one) and remove the 'space law [...] to you' part.

File: code/modules/mob/living/silicon/robot/robot.dm (hunk 3 above)
Comment (9600bauds) [reply code/modules/mob/living/silicon/robot/robot.dm:321]: Voxbloodhound

**Comment by gbasood** [comment]
@Intigracy @Probe1 pls

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 6]
```diff
@@ -318,12 +318,12 @@
...
 			module_sprites["Black Knight"] = "securityrobot"
-			module_sprites["Bloodhound"] = "bloodhound"
+			module_sprites["Hound"] = "bloodhound"
 			module_sprites["Securitron"] = "securitron"
-			module_sprites["Marina-SC"] = "marinaSC"
+			module_sprites["Marina-PK"] = "marinaSC"
 			module_sprites["#9"] = "servbot-sec"
 			module_sprites["Kodiak"] = "kodiak-sec"
-			to_chat(src, "<span class='warning'><big><b>Just a reminder, by default you do not follow space law, you follow your lawset</b></big></span>")
+			to_chat(src, "<font size='12' color='red'><b><b>You are expected to follow your laws above all else. You are not a Security Officer.</b></font>")
```
Comment (PJB3005) [inline code/modules/mob/living/silicon/robot/robot.dm:326]: No `<font>` tag. Use `danger` and `big` as classes.

**Comment by icantthinkofanameritenow** [comment]
Stopped caring, doesn't seem like it's necessary now.

**Pull request closed without merging**